doc/api/util.md

```diff
@@ -10,6 +10,67 @@ module developers as well. It can be accessed using:
 const util = require('util');
 ```
 
+## util.callbackify(original)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `original` {Function} An `async` function
+* Returns: {Function} a callback style function
+
+This method converts Promise returning API endpoints to ones that use callbacks.
```

Comment:
I open to change, but it's based on @benjamingr 's phrasing https://github.com/nodejs/node/pull/12712#discussion_r120115166
I think the idea is to give the motivation. Maybe:
`This utility is provided to help in adapting Promise-returning APIs to callback style.`